### ISSUE TITLE
Fulcro 3.4.17 breaks fulcro-css.localized-dom

### ISSUE THREAD

**Issue by MrEbbinghaus** [fulcrologic/fulcro#463]
With Fulcro 3.4.17 the following error appears:
3.4.16 work fine.

```
------ ERROR -------------------------------------------------------------------
/.m2/repository/com/github/awkay/workspaces/1.0.3/workspaces-1.0.3.jar!/nubank/workspaces/ui/cursor.cljs:70:3
--------------------------------------------------------------------------------
  67 |             (scroll-down options
  68 |                          (gobj/get this "container")
  69 |                          (gobj/get this (str "item-" (value->key value))))))}
  70 |   (dom/div :.container (dom-props (assoc props :ref #(gobj/set this "container" %)))
---------^----------------------------------------------------------------------
null
Wrong number of args (2) passed to: com.fulcrologic.fulcro-css.localized-dom/emit-tag
--------------------------------------------------------------------------------
```

I guess this is caused by https://github.com/fulcrologic/fulcro/commit/ebb5392eed3df03c7aabb4d23ff8056c644c4146 but I don't know the code well enough to provide a fix (at least for now).

**Comment by awkay** [comment]
Ugh, this is why I never touch the DOM generation code. That stuff is so hyper-optimized it's a land-mine.

**Comment by awkay** [comment]
Fixed in fulcro-garden-css. Released 3.0.9 for fulcro-garden-css to Clojars.
